# Static parameters fsm stand up for a1 real
standup:
  Kp_joint0: 20
  Kp_joint1: 15
  Kp_joint2: 15

  Kd_joint0: 3
  Kd_joint1: 3
  Kd_joint2: 3

  Kp_cartesian_x: 500
  Kp_cartesian_y: 500
  Kp_cartesian_z: 500

  Kd_cartesian_x: 5
  Kd_cartesian_y: 5
  Kd_cartesian_z: 5

#-------------------------------
static_params:
  max_vel_x: 0.7
  max_vel_y: 0.4
  max_turn_rate: 2.5

  Q_roll: 10
  Q_pitch: 10
  Q_yaw: 15
  Q_x: 1
  Q_y: 1
  Q_z: 30
  Q_w_roll: 0.5
  Q_w_pitch: 0.5
  Q_w_yaw: 3
  Q_vx: 10
  Q_vy: 10
  Q_vz: 5